4-WebApp-your-API/4-1-MyOrg/TodoListService/Controllers/TodoListController.cs

```diff
@@ -33,6 +34,7 @@ public TodoListController(IHttpContextAccessor contextAccessor)
 
         // GET: api/values
         [HttpGet]
+        [RequiredScope("ToDoList.Read")]
```

Comment:
>[RequiredScope("ToDoList.Read")] [](http://example.com/codeflow?start=8&length=32)

do we have a way to add check for the app permission ToDoList.Read.All ?

Reply:
Opened an [issue](https://github.com/AzureAD/microsoft-identity-web/issues/1641) for Jenny